modules/vlybyBidAdapter.js

```diff
@@ -0,0 +1,76 @@
+import { registerBidder } from '../src/adapters/bidderFactory.js'
+
+import { BANNER, VIDEO } from '../src/mediaTypes.js'
+import { config } from '../src/config.js';
+
+const ENDPOINT = '//europe-west3-vlybypoc2019.cloudfunctions.net/prebid';
+const BIDDER_CODE = 'vlyby';
+
+export const spec = {
+  code: BIDDER_CODE,
+  supportedMediaTypes: [VIDEO, BANNER],
+
+  isBidRequestValid: function (bid) {
+    if (bid && bid.params && bid.params.publisherId && bid.params.siteId) {
+      return true
+    }
+    return false
+  },
+
+  buildRequests: function (validBidRequests, bidderRequest = {}) {
+    const refererInfo = bidderRequest.refererInfo || {};
+    const gdprConsent = bidderRequest.gdprConsent || {};
+    const publisherDomain = config.getConfig('publisherDomain');
+    return {
+      method: 'POST',
+      url: `${ENDPOINT}`,
+      data: {
+        pubDomain: publisherDomain,
+        auctionId: bidderRequest.auctionId,
+        refererInfo: {
+          isAmp: refererInfo.isAmp,
+          numIframes: refererInfo.numIframes,
+          reachedTop: refererInfo.reachedTop,
+          referer: refererInfo.referer
+        },
+        gdprConsent: {
+          consentString: gdprConsent.consentString,
+          gdprApplies: gdprConsent.gdprApplies
+        },
+        bidRequests: validBidRequests.map(({ params, sizes, bidId, adUnitCode }) => ({
+          bidId,
+          adUnitCode,
+          params,
+          sizes
```

Comment:
What's the effect of the `sizes` being submitted. Is this only for `BANNER` ads or does the player take them into account too?

Reply:
sizes and Banner are special case. For some Publisher we will provide Banner with sizes like 300x250 or 300x600